addons/event/views/event_registration_views.xml

```diff
@@ -180,6 +152,7 @@
                                     <div class="oe_kanban_content h-100">
                                         <div class="o_kanban_record_body h-100 d-flex flex-column">
                                             <b class="o_kanban_record_title"><field name="name"/></b>
+                                            <field name="state" widget="badge" decoration-success="state == 'done'" class="position-absolute top-0 end-0 o_event_registration_kanban_badge"/>
```

Comment:
I see there can be some slight overlap with details for certain screen sizes (if details occupy a long space). Can you have a quick look? 
![image](https://github.com/user-attachments/assets/17bc5967-f201-41f2-afa5-f4fe2340959e)